.github/workflows/integration.yaml

```
@@ -111,7 +111,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.11']
+        python-version: ['3.11']
```

```suggestion
        python-version: ['3.8', '3.11']
```
We want to test it in the oldest and newest versions we support